### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by tomrsae**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+#   push:
+#     branches: [ master ]
+# unnecessary?
```
Comment (ngorke) [inline .github/workflows/ci.yml]: yes

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Run tests
+      run: cargo test --verbose
```
Comment (ngorke) [inline .github/workflows/ci.yml]: remove verbose? 🧅

**Review by ngorke — CHANGES_REQUESTED** [review]
🧅 👍

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tomrsae) [reply .github/workflows/ci.yml]: Resolved in most recent commit.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (tomrsae) [reply .github/workflows/ci.yml]: Resolved in most recent commit.

**Review by ngorke — APPROVED** [review]
🧅 🌮

**Pull request merged**